app/models/disease.py

```
@@ -0,0 +1,39 @@
+
+class Disease(Base):
+    __tablename__ = "diseases"
+    
+    name: Mapped[str] = mapped_column(String(100), index=True)
+    
+    details: Mapped[List["DiseaseDetail"]] = relationship(
+        back_populates="disease",
+        cascade="all, delete-orphan",
+        lazy="selectin"
+    )
+    hospitals: Mapped[List["Hospital"]] = relationship(
```

Ruff (0.8.2)

Undefined name `Hospital`

(F821)